server.ts

```diff
@@ -170,11 +182,17 @@ app.post("/conferences", async (req, res) => {
     const slug = slugify(name).toLowerCase();
     console.log("using slug", slug);
     if (slugsToIds.has(slug)) {
-      res.status(409).send();
-      return;
+      let conferenceId = slugsToIds.get(slug)!;
+      try {
+        await getConference(conferenceId);
+        res.status(409).send();
+        return;
+      } catch (e) {
+        slugsToIds.delete(slug);
```

Comment:
If I understand correctly the fix that you delete the slug mapping here, and on line 218 when the mapping returned a conference Id that no longer exists? https://github.com/Bandwidth/webrtc-sample-conference-node/pull/19/files#diff-5c092da67283b0b787b618509fa7b96fR218